### PR TITLE
add default prompts to edit profile

### PR DESCRIPTION
**Pull request by ItsiW**
**Frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [x] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/frontend/src/features/profile/edit/constants.ts [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
 <br>
+e.g. I think couch surfing should be free...
+<br>
+# My favorite travel story
+<br>
+e.g. One time I...
 `;
 
 export const DEFAULT_HOBBIES_HEADINGS = `# Art
+
```
Comment (aapeliv) [inline app/frontend/src/features/profile/edit/constants.ts]: Missing `<br>`

**Review by aapeliv — CHANGES_REQUESTED** [review]
The "e.g." isn't that pleasing, but 🤷‍♂️

**Comment by aapeliv** [comment]
Also I think it would be good to give better examples. I personally find it a bit lame when people just list bands/movies/etc in their free time section

**Comment by ItsiW** [comment]
> The "e.g." isn't that pleasing, but man_shrugging

This is intentional. It needs to look a little bad so it prompts people to delete the line. 

Could you provide some examples of desired text? I've mostly only ever seen lists, I tend to not fill these sections at all.

**Review by aapeliv — APPROVED** [review]
LGTM

**Pull request merged**